src/main/java/com/segment/analytics/android/integrations/amplitude/AmplitudeIntegration.java

```diff
@@ -54,6 +54,8 @@ public String key() {
   boolean trackCategorizedPages;
   boolean trackNamedPages;
   boolean useLogRevenueV2;
+  boolean enableLocationListening;
```

Comment:
This does not need to be a member instance. It can be a local field like apiKey or trackSessionEvents